regolith/builders/preslistbuilder.py

```diff
@@ -31,12 +31,31 @@ def construct_global_ctx(self):
         gtx['str'] = str
         gtx['zip'] = zip
 
+    def list_group_members(self, grp):
+        # get all group members
+        grpmembers = []
+        print(self.gtx['people'][0]['_id'])
+        for person in self.gtx['people']:
+            for position in person.get('education', {}):
+                if position.get('group', None) == grp:
+                    if person['_id'] not in grpmembers:
+                        grpmembers.append(person['_id'])
+            for position in person.get('employment', {}):
+                if position.get('group', None) == grp:
+                    if person['_id'] not in grpmembers:
+                        grpmembers.append(person['_id'])
+        return grpmembers
+
+
     def latex(self):
         """Render latex template"""
         for group in self.gtx['groups']:
             pi = fuzzy_retrieval(self.gtx['people'], ['aka', 'name', '_id'],
                                  group['pi_name'])
-
+        listgrpmembers = self.list_group_members('bg')
```

Comment:
We generally don't put type names into varaible names.  I suggest renaming to just `grpmembers`

Reply:
I would pull this from the `regolithrc` since there is a `groupname` field. This can be accessed by `rc.groupname`.

Reply:
OK, I ran into a problem with this.  If we want to iterate over more than one group (e.g., columbia and BNL groups) I would need to run the program twice with a manual edit of the rc file in between.

Also, this bifurcates the definitive place where groups are specified.  We have a ``groups.yml`` which is the groups we are tracking, and an rc.groupname which is some name/tag that is specified in the rc.  These two sources of truth don't talk to each other.   For example, the group specified in ``rc.groupname`` may not even exist in groups.yml.  I would rather be able to specify a group id when I run the program, so I could do things like ``[groupnames.append(list_group_names(grp)) for grp in ['bg','bnl']]`` (or somesuch that keeps the list items unique.)
Is there a reason to not do this and to use the rc instead?